source/change-streams/tests/unified/change-streams.yml

```diff
@@ -70,3 +73,63 @@ tests:
             ]
           }
         }
+
+  - description: "Test with document comment"
+    runOnRequirements:
+      - minServerVersion: "4.4"
+    operations:
+      - name: createChangeStream
+        object: *collection0
+        arguments:
+          pipeline: []
+          comment: &comment0 { name: "test1" }
+        saveResultAsEntity: &changeStream0 changeStream0
+    expectEvents:
+      - client: *client0
+        events:
+          - commandStartedEvent:
+              command:
+                aggregate: *collection0
+                pipeline:
+                  - $changeStream: {}
+                comment: *comment0
+
+  - description: "Test with document comment - pre 4.4"
+    runOnRequirements:
+      - minServerVersion: "3.6.0"
+        maxServerVersion: "4.2.99"
+    operations:
+      - name: createChangeStream
+        object: *collection0
+        arguments:
+          pipeline: []
+          comment: &comment0 { name: "test1" }
+        expectError:
+          isClientError: false
+    expectEvents:
+      - client: *client0
+        events:
+          - commandStartedEvent:
+              command:
+                aggregate: *collection0
+                pipeline:
+                  - $changeStream: {}
+                comment: *comment0
+
+  - description: "Test with string comment"
```

Comment:
I assume there's no need to duplicate both the document _and_ comment tests for pre 4.4 servers? I see other test files duplicate the "string" version while this one duplicates the "document" version.

Reply:
For change streams a string comment would work down to 3.6. Hence the "document" version for pre 4.4 servers here. I'll add `minServerVersion` to the "string" version to make things more clear here.